docs/transpile/representing_quantum_computers.ipynb

```diff
@@ -74,7 +84,7 @@
    "source": [
     "### Transpile the circuit to `backend` target\n",
     "\n",
-    "This example uses the default `optimization_level=1` to transpile to the `backend` `target`, which providers all the information to the transpiler that is necessary to convert circuit to one that will run on the system."
+    "This example uses the default `optimization_level=1` to transpile to the `backend`'s `target`, which providers all the information to the transpiler that is necessary to convert circuit to one that will run on the system.  (Note that if a `target` is specified in the `transpile` function, this will override the target from the `backend`.)"
```

Comment:
```suggestion
    "This example uses the default `optimization_level=1` to transpile to the `backend`'s `target`, which provides all the information needed to convert circuit to one that will run on the system. (Note that if a `target` is specified in the `transpile` function, this will override the target from the `backend`.)"
```